GVFS/GVFS.Common/MountInternal.cs

```diff
@@ -0,0 +1,23 @@
+﻿using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Text;

```

Comment:
Remove unnecessary using statements

Reply:
Done, but it's a little spooky you could spot that from just a PR :-)